src/libraries/EIP712Hash.sol

```diff
@@ -8,6 +8,10 @@ import {EIP712Types} from 'src/libraries/types/EIP712Types.sol';
 /// @author Aave Labs
 /// @notice Helper methods to hash EIP712 typed data structs.
 library EIP712Hash {
```

Comment:
Happy to see this back in the common libraries :)

Reply:
not a fan, i d prefer to keep it separate just because it does not make sense to have a library with eip712 hashes about vaults in a posm contract